libs/Daux.php

```diff
@@ -90,6 +93,18 @@ public function initializeConfiguration($override_file = 'config.json')
         }
     }
 
+    public function setThemesPath($path)
+    {
+        $this->themes_path = $path;
+        if (!is_dir($this->themes_path) &&
+            !is_dir($this->themes_path = $this->local_base . DIRECTORY_SEPARATOR . $this->themes_path)
+        ) {
+            throw new Exception('The Themes directory does not exist. Check the path again : ' . $this->themes_path);
+        }
+        $this->options['themes_path'] = $this->themes_path;
+        $this->options['templates'] = $this->themes_path . DIRECTORY_SEPARATOR . 'core' . DIRECTORY_SEPARATOR . 'templates';
```

Comment:
This "templates" concept should be removed and the {themes_path}/core/templates/ should just be assumed.